### PR TITLE
search-blitz: generated queries for dogfood

### PR DESCRIPTION
**Pull request by keegancsmith**
We want to run search blitz on dogfood. However, we need to adjust the queries since we want to target the gigarepo instead of the megarepo. This commit adds logic to generate the queries from the cloud queries.

Why this approach:
- we test the output and can assert megarepo is not called.
- we can inspect the generated output in PRs.
- when we update queries_cloud.txt we will be told to think about dogfood.
- go test is already needed for the above, so can just use it instead of go generate.

Test Plan: unit tests

### CODE REVIEW

File: internal/cmd/search-blitz/config.go [hunk 1]
```diff
@@ -2,15 +2,16 @@ package main
...
 	"strings"
 	"time"
 
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
-//go:embed queries.txt
-var queriesRaw []byte
+//go:embed queries_*.txt
+var queriesFS embed.FS
```
Comment (stefanhengl) [inline internal/cmd/search-blitz/config.go:14]: TIL 🎉

**Review by stefanhengl — APPROVED** [review]
Nice!

One question. We target gigarepo on dogfood and megarepo on production? Why don't we target both on dogfood?

**Comment by keegancsmith** [comment]
> One question. We target gigarepo on dogfood and megarepo on production? Why don't we target both on dogfood?

We could, but I'd have to change my approach in the PR. WDYT, should we target both? I can adjust this PR to take into account this use case. I can't think of any big downsides of doing both on dogfood.

**Comment by stefanhengl** [comment]
> > One question. We target gigarepo on dogfood and megarepo on production? Why don't we target both on dogfood?
> 
> We could, but I'd have to change my approach in the PR. WDYT, should we target both? I can adjust this PR to take into account this use case. I can't think of any big downsides of doing both on dogfood.

I think it would be good to target both on dogfood because it would be nice to compare the numbers based on the same environment.

**Comment by keegancsmith** [comment]
Will send out another PR soon.

**Pull request closed without merging**